p/scripts/user.query.js

```diff
@@ -11,27 +11,63 @@ function init_draggable() {
 		return;
 	}
 	
-	let source;
+	var source;
 	const configureQueries = document.querySelector('#configureQueries');
+	const addMarker = function(position, element) {
```

Comment:
Personal preference: I like a space before the parenthesis for anonymous functions. But as you are using arrow function syntax, you might just as well use it here